python/pyspark/ml/feature.py

```diff
@@ -2629,8 +2629,21 @@ class ChiSqSelector(JavaEstimator, HasFeaturesCol, HasOutputCol, HasLabelCol, Ja
     """
     .. note:: Experimental
 
-    Chi-Squared feature selection, which selects categorical features to use for predicting a
-    categorical label.
+    Creates a ChiSquared feature selector.
+    The selector supports different selection methods: `numTopFeatures`, `percentile`, `fpr`,
+    `fdr`, `fwe`.
+    `numTopFeatures` chooses a fixed number of top features according to a chi-squared test.
+    `percentile` is similar but chooses a fraction of all features instead of a fixed number.
+    `fpr` chooses all features whose p-value is below a threshold, thus controlling the false
+    positive rate of selection.
+    `fdr` uses the [Benjamini-Hochberg procedure]
```

Comment:
```
`Benjamini-Hochberg procedure <https://en.wikipedia.org/wiki/False_discovery_rate#Benjamini.E2.80.93Hochberg_procedure>`_
```
This is the Python style of using doc links. You can refer https://github.com/apache/spark/blob/master/python/pyspark/ml/regression.py#L1308